database/insertData/06_applications_A_featureShowcase.js

```diff
@@ -17,36 +17,43 @@ exports.queries = [
             create: [
               {
                 id: 1000
+                status: DRAFT 
                 value: { text: "Craig" }
                 templateElementToTemplateElementId: { connectById: { id: 1001 } }
               }
               {
                 id: 1001
+                status: DRAFT 
                 value: { text: "Drown" }
                 templateElementToTemplateElementId: { connectById: { id: 1002 } }
               }
               {
                 id: 1002
+                status: DRAFT 
                 value: { text: "c_drown" }
                 templateElementToTemplateElementId: { connectById: { id: 1004 } }
               }
               {
                 id: 1003
+                status: DRAFT 
                 value: { text: "craig@sussol.net" }
                 templateElementToTemplateElementId: { connectById: { id: 1005 } }
               }
               {
                 id: 1004
+                status: DRAFT 
                 value: { text: "123456" }
                 templateElementToTemplateElementId: { connectById: { id: 1006 } }
               }
               {
                 id: 1005
+                status: DRAFT 
                 value: { optionIndex: 0, text: "Manufacturer" }
                 templateElementToTemplateElementId: { connectById: { id: 1007 } }
               }
               {
                 id: 1006
+                status: DRAFT 
```

Comment:
Doesn't really matter, but since you added a default value for this, we don't really need to specify "DRAFT".

Reply:
2810c55830694ef2560b2d621bcb001f20f2009d